pkg/ice/agent.go

```diff
@@ -711,6 +711,15 @@ func (a *Agent) handleInbound(m *stun.Message, local *Candidate, remote net.Addr
 	}
 }
 
+// NoSTUNSeen processes non STUN traffic from a remote candidate
+func (a *Agent) NoSTUNSeen(local *Candidate, remote net.Addr) {
```

Comment:
I would keep this private:
```suggestion
func (a *Agent) noSTUNSeen(local *Candidate, remote net.Addr) {
```